app/scripts/views/sign_up.js

```diff
@@ -35,6 +38,17 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
     return null;
   }
 
+  var NO_OF_HASHING_FUNCTIONS = 8;
+  var bloom = null;
+  $.ajax({
```

Comment:
Let's use [xhr.ajax](https://github.com/mozilla/fxa-content-server/blob/9ec6b1812e76fa9576462bd9c4cebf4544698f9f/app/scripts/lib/xhr.js#L30) instead, that's our wrapper so that we don't have to assume jQuery and allows us to mock in requests/responses in the tests w/o modifying the jQuery object.